extensions/vscode/webviews/homeView/src/components/EvenEasierDeploy.vue

```diff
@@ -95,10 +82,10 @@
     <vscode-button
       v-else
       class="w-full add-deployment-btn"
-      @click="onAddDeployment"
+      @click="onSelectDeployment"
       data-automation="add-deployment-button"
     >
-      Add Deployment
+      Select Deployment...
```

Comment:
Simplified ux. Now we either show the currently selected deployment (in the way we have done before) or we show this new simple button. This is the biggest change which allows us to eliminate the upfront recursive query before the user can interact with the UX.

Reply:
The fact that this is sometimes a button and sometimes a dropdown menu is a little jarring. I see why that's helpful (the button doesn't have to present any context) but I wonder if we can unify these somehow, maybe we could show the dropdown with the "Get deploying" message here?